src/commands/createNewProject/validateFunctionProjects.ts

```diff
@@ -79,7 +80,7 @@ function isInitializedProject(folderPath: string): boolean {
  * JavaScript debugging in the func cli had breaking changes in v2.0.1-beta.30 (~6/2018). This verifies users are up-to-date with the latest working debug config.
  * See https://aka.ms/AA1vrxa for more info
  */
-async function verifyDebugConfigIsValid(projectLanguage: string | undefined, folderPath: string, actionContext: IActionContext): Promise<void> {
+async function verifyJSDebugConfigIsValid(projectLanguage: string | undefined, folderPath: string, actionContext: IActionContext): Promise<void> {
```

Comment:
I see why you're doing this, but it feels kind of messy to me to verify that the projectLanguage is the respective language when the functions are named for each language.  Why not just have the switch case within a general `verifyDebugConfigIsValid` function?

Reply:
On second thought, I guess there's a lot of custom logic.